### ISSUE TITLE
Create KDE Application Launcher entry for Cemu

### ISSUE THREAD

**Issue by Godsbane** [dragoonDorise/EmuDeck#195]
had a request. makes sense if we can.

@AngelofWoe

**Comment by AngelofWoe** [comment]
In `/home/deck/.local/share/applications`, create `proton-launch.sh.desktop`:
```ini
[Desktop Entry]
Exec=/run/media/mmcblk0p1/Emulation/tools/proton-launch.sh
MimeType=application/x-ms-dos-executable;
Name=proton-launch.sh
NoDisplay=true
Type=Application
```
In `$HOME/.config/mimeapps.list`, under `[Added Associations]` add:
application/x-ms-dos-executable=proton-launch.sh.desktop;

**Comment by AngelofWoe** [comment]
For Cemu in KDE Plasma's Application Launcher -

Create `/home/deck/.local/share/applications/Cemu.desktop`, containing:
```ini
[Desktop Entry]
Name=Cemu
Comment=Run Cemu through Proton
Exec=/usr/bin/bash /run/media/mmcblk0p1/Emulation/tools/launchers/cemu.sh
Icon=?
Terminal=false
Type=Application
Categories=Game;
```

And make it executable.

**Comment by AngelofWoe** [comment]
We got a request to add a choice to have all of our shortcuts we add to the desktop to be on the desktop, in the application launcher, or both, so we can use a similar choice flag.

I recommend we make a function to create shortcuts and have it have a flag for desktop, launcher, or both, and then just give the function the data it needs.

**Comment by dragoonDorise** [comment]
Fixed in 0.17.6
